src/client/ui/hierarchy/HierarchyPanelContainer.js

```diff
@@ -114,7 +114,7 @@ class HierarchyPanelContainer extends Component {
           className: classNames("node", {
             "is-active": this.props.editor.selected && node.object.id === this.props.editor.selected.id
           }),
-          onMouseDown: e => this.onMouseDownNode(e, node)
+          onMouseUp: e => this.onMouseUpNode(e, node)
```

Comment:
This is the extra bug fix I snuck in and referred to in the PR. It directly affects renaming nodes and it's small enough that I don't feel like it needs to go in another PR 🤷‍♂️